common/reviews/api/rush-lib.api.md

```diff
@@ -413,10 +413,15 @@ export class RushConfigurationProject {
     // @internal
     constructor(projectJson: IRushConfigurationProjectJson, rushConfiguration: RushConfiguration, tempProjectName: string);
     get cyclicDependencyProjects(): Set<string>;
+    // @deprecated
     get downstreamDependencyProjects(): string[];
+    get downstreamDependencyProjectSet(): Set<string>;
```

Comment:
Should it be a `ReadonlySet`?

Reply:
Can't, because `RushConfiguration` reads and mutates it.